lib/ReactViews/Generic/dropdown.scss

```diff
@@ -21,6 +21,7 @@
 
   &.is-open {
     transform: scaleY(1);
+    overflow-y: scroll;
```

Comment:
Is there any particular reason you picked `overflow-y: scroll` over `overflow-y: auto`?

Reply:
Short answer no. 

In both chrome & firefox on mac the scrollbar disappears when not hovered over the element which is the same behaviour as auto I think. IE11 might be more finicky and apply a scrollbar even if it's not required.

Reply:
should use auto, no need to constantly have a scroll bar

Reply:
Updated